hedera-node/hedera-consensus-service-impl/src/test/java/module-info.java

```diff
@@ -2,7 +2,13 @@
     requires com.hedera.node.app.service.consensus;
     requires com.hedera.node.app.service.consensus.impl;
     requires org.junit.jupiter.api;
+    requires com.hedera.node.app.service.mono.testFixtures;
+    requires com.hedera.node.app.service.mono;
+    requires org.mockito;
+    requires org.mockito.junit.jupiter;
+    requires org.assertj.core;
+    requires com.google.protobuf;
 
-    opens com.hedera.node.app.service.consensus.impl.test to
-            org.junit.platform.commons;
+    opens com.hedera.node.app.service.consensus.impl.test;
+    opens com.hedera.node.app.service.consensus.impl.handlers.test;
```

Comment:
change to `open module com.hedera.node.app.service.consensus.impl.test {...}` and you can get rid of all this lines.

Reply:
Ah I finally understand, thank you